linera-core/src/chain_worker/state/mod.rs

```diff
@@ -469,11 +469,13 @@ where
                 bundle_vecs.push((medium, bundles));
             }
         }
-        Ok(CrossChainRequest::UpdateRecipient {
-            sender: self.chain.chain_id(),
-            recipient,
-            bundle_vecs,
-        })
+        Ok(
+            (!bundle_vecs.is_empty()).then(|| CrossChainRequest::UpdateRecipient {
+                sender: self.chain.chain_id(),
+                recipient,
+                bundle_vecs,
+            }),
```

Comment:
Please remove the unnecessary `,`. Personally I would prefer a basic syntax `if bundle_vecs.is_empty() then { Ok(None) } else { Ok(Some(..)) }` (removing the negation in the process)